.github/workflows/ci.yml

```diff
@@ -186,10 +186,10 @@ jobs:
           git tag "v$CURRENT_VERSION"
           git push origin "v$CURRENT_VERSION"
```

Comment:
If moving publishing to a workflow, I think it's better to move the tagging into that workflow too. For bonus points, maybe tag the version after `cargo publish` rather than before.

Reply:
Both `taiki-e/create-gh-release-action@v1` and `taiki-e/upload-rust-binary-action@v1` must run on the tag ref.

I tried to set a enviormental variable and use it later on, but this somehow failed on Windows. 

Actually, the pair programming sounds like a cool idea, as I have spent some time already on trial and error with moderate success.

Reply:
It seems like they would accept a `ref` argument though:
https://github.com/taiki-e/create-gh-release-action/blob/main/action.yml#L45-L49
https://github.com/taiki-e/upload-rust-binary-action/blob/main/action.yml#L61-L66

If we `git tag` and push the tag before running those actions, and set their `ref` to the tag name, I think that should work.

Let me know over Zulip what times are best for pair-programming for you in the next few days, and we can make something work.